metadata.rb

```diff
@@ -7,6 +7,6 @@
 issues_url 'https://github.com/ncerny/chef_stack/issues' if respond_to?(:issues_url)
 source_url 'https://github.com/ncerny/chef_stack' if respond_to?(:source_url)
 
-version '0.8.6'
+version '0.9.0'
```

Comment:
I would suggest a patch release for the deprecation warnings. Users pinning to `0.8` will not see it. I also recommend the follow up patch release that truly deprecates this cookbook so it can't be used unless pinned specifically to `0.8.6`.

Maybe fully deprecate in 30 days or so?